Bin/Documentation.php

```diff
@@ -309,7 +316,40 @@ public function main()
             }
         }
 
-        echo "\n", 'Open file://', $workspace, '/index.html', '.', "\n";
+        $pathname = escapeshellarg('file://' . $workspace . '/index.html');
+
+        echo "\n";
+
+        if (true === $open) {
+            if (isset($_SERVER['BROWSER'])) {
```

Comment:
You are right, this kind of over engineering is never the right answer 😄. It's better to discover problems through issues when they will be found and documented...